### PR TITLE
Add demo support packet

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Demonstrates how a plugin can include it's own support packet content with the new API feature.

resim

### CODE REVIEW

**Comment by DHaussermann** [comment]
@isacikgoz when I deploy this the UI is working and the demo plugin is rendering the extra check-box. Just to confirmed the scope of this PR is only the check box in the UI correct?

Checking the box does not add any contents to the support packet for me.

File: plugin.json [hunk 1]
```diff
@@ -141,5 +141,8 @@
                 "default": 0
             }
         ]
+    },
+    "props": {
+        "support_packet": "Demo plugin support packet"
```
Comment (hanzei) [inline plugin.json:146]: At some point, we should make this a proper field in the manifest. For now, this is fine.

**Review by hanzei — APPROVED** [review]
Thanks Ibrahim!

File: plugin.json (hunk 1 above)
Comment (isacikgoz) [reply plugin.json:146]: Yeah, if we have it's own field I think the plugin won't be backwards compatible hence I went with this.

**Comment by isacikgoz** [comment]
> @isacikgoz when I deploy this the UI is working and the demo plugin is rendering the extra check-box. Just to confirmed the scope of this PR is only the check box in the UI correct?
> 
> Checking the box does not add any contents to the support packet for me.

@DHaussermann We need to check the server logs but probably something was off due to dependencies. It works for me locally but I have overrides for dependencies. I'll update the dependency and ask a second round of review from you.

File: plugin.json (hunk 1 above)
Comment (hanzei) [reply plugin.json:146]: What part of the code would suffer form from a backwards compatible issue?

Comment (isacikgoz) [reply plugin.json:146]: I mean the schema of the manifest needs to be updated so it whomever going to parse the new manifest would require a newer version of the code. Haven't tested though..

**Comment by isacikgoz** [comment]
I updated the server dependency and it should be fine for now, can you try again once you get a chance @DHaussermann, thanks!

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Confirmed that check box for demo support is correctly shown in the UI
- Confirmed that checking the box will now include a demo file
image

LGTM!

Thanks @isacikgoz

**Pull request merged**